src/cooldown.ts

```diff
@@ -0,0 +1,14 @@
+import { Collection, Snowflake } from 'discord.js';
+import config from './config';
+
+const collection = new Collection<Snowflake, number>();
+
+export default {
+    isCooled: (id: Snowflake): boolean => {
+        const nextCommand = collection.get(id);
+        return nextCommand ? nextCommand > Date.now() : false;
```

Comment:
what do you think of having the same condition as before?
```suggestion
        return nextCommand && nextCommand > Date.now();
```

Reply:
I went with the ternary because of a previous suggestion someone made on this or on the other PR, but that looks even better

Reply:
ahh forgot about that, thanks for explaining 👍